src/App.js

```diff
@@ -1,13 +1,13 @@
-import logo from "./logo.svg";
-import "./App.css";
-
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>Learn react in the gym</p>
-      </header>
+    <div className="flex">
```

Comment:
I would suggest to avoid using `flex` as a class. It can easily be confused with `display: flex` or when using a CSS library like TailwindCSS which has `flex` as a class. You could name it something like `button-container`